This allows us to source rabbitmq-env from a script that doesn't live in
the same directory.

--- rabbitmq-server-2.8.7/scripts/rabbitmq-env	Thu Sep 27 04:08:40 2012
+++ rabbitmq-server-2.8.7/scripts/rabbitmq-env	Mon Nov  5 11:10:09 2012
@@ -32,7 +32,7 @@
     fi
 done
 
-SCRIPT_DIR=`dirname $SCRIPT_PATH`
+SCRIPT_DIR=${SCRIPT_DIR:-`dirname $SCRIPT_PATH`}
 RABBITMQ_HOME="${SCRIPT_DIR}/.."
 [ "x" = "x$HOSTNAME" ] && HOSTNAME=`env hostname`
 NODENAME=rabbit@${HOSTNAME%%.*}
